src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person
+};
+
+export const PersonLink: FC<Props> = ({
```

Comment:
It's better to destructure the `Props` directly in the function argument instead of inside the function body. Change `({ person })` to `({ person: { slug, sex, name } })` and remove lines 13 and 14.